### PR TITLE
Added umd support.

### PR DESCRIPTION
**Pull request by nkovacic**
Removed global angular from every module (window.angular).
Added gulp-umd.
Added main field to package.json

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -26,7 +27,7 @@ gulp.task('watch', function() {
 });
 
 gulp.task('webserver', function() {
-	webserverInstance = gulp.src('./dev').pipe(webserver({host: '0.0.0.0'}));
+	webserverInstance = gulp.src('./dev').pipe(webserver({ host: '127.0.0.1', port: 8000 }));
```
Comment (brunokrebs) [inline gulpfile.js]: Host: '0.0.0.0' makes webserver listen to all interfaces, not only localhost. So it is possible to test it with other browsers/OS. Why have you changed this?

File: gulpfile.js [hunk 2]
```diff
@@ -51,7 +52,7 @@ gulp.task('sass', function() {
 
 gulp.task('minify', ['sass'], function() {
 	return gulp.src(getDestination() + '/editor.css')
-		.pipe(gulpif(development == false, rename(renameMin)))
+		.pipe(gulpif(development === false, rename(renameMin)))
```
Comment (brunokrebs) [inline gulpfile.js]: Why are you changing from loose to strict comparisons?

Comment (nkovacic) [inline gulpfile.js]: I am using jshint and it displays warnings for non-strict comparisons. I can change them back to the original comparisons.

File: gulpfile.js (hunk 1 above)
Comment (nkovacic) [inline gulpfile.js]: You are absolutely right. I changed it back to the original host.

**Pull request merged**